templates/tutorialv2/includes/summary.part.html

```diff
@@ -64,7 +66,9 @@ <h4 data-num="{{ child.position_in_parent|roman }}">
             {% else %}
                 <ol>
                     {% for subchild in child.children %}
-                        <li {% if current_container.long_slug == subchild.long_slug %}class="current"{% endif %}>
+                        <li class="{% if current_container.long_slug == subchild.long_slug %}current{% endif %}
+                            {% if not child.ready_to_publish %} not-ready{% endif %}"
```

Comment:
Pour une raison qui m'échappe, ça ne fait pas ce que je pense que c'est sensé faire:

![screenshot from 2018-08-07 18-37-49](https://user-images.githubusercontent.com/7889675/43789817-37ca56ca-9a71-11e8-9a35-e9c9b0c4fb0e.png)

Reply:
Je sais pourquoi, j'ai pas de CSS pour les `not-ready` dans un `li`.

Reply:
Oui, mais il me semble qu'en inspectant le code, les classes n'y étaient pas non plus. M'enfin je te  laisse investiguer :)

Reply:
souvenez-vous que seul les parties et chapitre ont la marque "not ready", les sections ne l'ont pas.

Reply:
On est d'accord que toujours pas (sur la derbière version en date), ou je sais pas interpréter le code convenablement ? J'ai bien `make build-front` (mais comme on le voit sur la capture, la classe n'est pas mise)

![screenshot from 2018-08-08 19-37-58](https://user-images.githubusercontent.com/7889675/43854280-b7936ba4-9b42-11e8-8679-5b923fee5c79.png)

Reply:
Nan, j'ai rien dit, je sais pas lire ^^

Reply:
C'est pour ça que j'ai modifié en `not child.ready_to_publish and not child.text`. D'ailleurs, `child.parent` renvoie quoi ? Parce que j'ai testé `child.parent.ready_to_publish` sur un extrait qui appartient à un chapitre prêt à être publié, et ça me renvoie `False`...